tensorflow/examples/android/src/org/tensorflow/demo/CameraActivity.java

```diff
@@ -62,6 +87,89 @@ protected void onCreate(final Bundle savedInstanceState) {
     }
   }
 
+  /**
+   * Callback for android.hardware.Camera API
+   */
+  @Override
+  public void onPreviewFrame(final byte[] bytes, final Camera camera) {
+    if (computing) {
+      return;
+    }
+    computing = true;
+    Camera.Size previewSize = camera.getParameters().getPreviewSize();
+    yuvBytes[0] = bytes;
+    try {
+      // Initialize the storage bitmaps once when the resolution is known.
+      if (previewSize.width !=0 ||  previewSize.height !=0) {
+        previewHeight = previewSize.height;
+        previewWidth = previewSize.width;
+        rgbBytes = new int[previewWidth * previewHeight];
```

Comment:
@osdamv You could add a check to see if rgbBytes == null before initializing. Right now it looks like onPreviewSizeChosen will be called every frame.